bluebrain/repo-bluebrain/packages/py-entity-management/package.py

```diff
@@ -9,11 +9,10 @@
 class PyEntityManagement(PythonPackage):
     """Pythonic Blue Brain Nexus access library."""
 
-    homepage = "https://bbpgitlab.epfl.ch/nse/entity-management"
-    git = "ssh://git@bbpgitlab.epfl.ch/nse/entity-management.git"
+    homepage = "https://github.com/BlueBrain/entity-management"
+    git = "https://github.com/BlueBrain/entity-management.git"
 
-    version("1.2.44", tag="entity-management-v1.2.44")
-    version("1.2.41", tag="entity-management-v1.2.41")
+    version("1.2.46", tag="v1.2.46")
```

Comment:
(using the `pypi = ` key and a proper sha is the preferred way to deal with Python packages.

Reply:
The only reason I preferred this way is because it is a little cleaner and symmetric with how we do it with gitlab. However, if it's preferable to specify the pypi distribution, I'll switch to that.